src/lib.rs

```diff
@@ -161,20 +201,25 @@
 //! As mention above, rspotify uses `maybe_async` crate to switch between async
 //! and blocking client, which is triggered inside `Cargo.toml`, so there is
 //! something you need to pay attention to when you are trying to build `rspotify`
+//!
+//! Build with `client-reqwest` feature, the `async` version, and this would compile Rspotify with [`reqwest`](https://docs.rs/reqwest/)
```

Comment:
AFAIK these build instructions are only for `rspotify` itself. Are they meant for developers of this library? In that case I think they'd fit better in the README, as it's only hosted on GitHub